app/controllers/ems_common.rb

```diff
@@ -385,6 +441,10 @@ def button
       when "network_router_tag"               then tag(NetworkRouter)
       when "orchestration_stack_tag"          then tag(OrchestrationStack)
       when "security_group_tag"               then tag(SecurityGroup)
+      # Physical server
+      when "physical_server_refresh"          then refreshphysicalservers
+      when "physical_server_delete"           then deletephysicalservers
+      when "physical_server_edit"             then editphysicalservers
```

Comment:
I would expect this `button` handling in `physical_server_controller`, not here in `ems_common`